src/main/java/Duke.java

```diff
@@ -1,10 +1,32 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Hello! I'm Duke\n     What can I do for you?");
+        System.out.println("    ____________________________________________________________");
```

Comment:
I suggest you keep those lines _____ in a private static final variable, so you can reuse them easily.